README.md

```diff
@@ -165,24 +165,38 @@ When you have installed all the desired plugins, please run this commands `npm r
 
 We are using [Leto-Modelizer-Api](https://github.com/ditrit/leto-modelizer-api) authentication (as a backend). To configure it, you must fill the configuration inside `global.config.json` root file.
 
+We have administration view [Leto-Modelizer-Admin](https://github.com/ditrit/leto-modelizer-admin), for user who have `admin` role. To manage users, groups and roles.
```

Comment:
for users who have `admin` role (to manage users, groups and roles).
OR
for users who have `admin` role, to manage users, groups and roles.

Reply:
done